### ISSUE TITLE
ColumnHeaderRenderer

### ISSUE THREAD

**Issue by Stubsz** [andrglo/react-window-grid#3]
I have used the column header renderer, but this seems to spit out my div without any of the absolute positioning applied which means that the headers are showing beneath each other instead of in the correct position.

Are you able to provide some clarity on this please, as I'd like to be able to have some CSS control over the display of the column headers, and in the future, maybe allow an onClick event to take place, but at the moment I am unable to use the renderer with the headers appearing within the wrong place when using it.

Thanks

**Comment by Stubsz** [comment]
I think the problem was that the renderer wasn't passing the style prop, so looks like it is now working, but your NPM documentation might need to clarify this in the "Type" column so that we know to pass it to the signature

**Comment by andrglo** [comment]
Fixed, thanks!

**Comment by Stubsz** [comment]
Hi there,

I have also found an issue whereby if the horizontal scrollbar is showing, then the vertical scroll is misaligned with the sticky first column when you scroll to the bottom – it looks like it isn’t taking into account the height of the bottom scrollbar to offset it.

Any ideas on how to fix this?

Thanks

David Stubbins
Software Engineer

E:  david.stubbins@proagrica.com<mailto:david.stubbins@proagrica.com>
W: www.proagrica.com<http://www.proagrica.com/>
T:  +44 (0)1594 545003

[cid:image001.png@01D1E1AE.32E9FE90]

**Comment by Stubsz** [comment]
Ignore me.

I think it’s because I set the columnWidths in useEffect, so I need to change where that’s set so that it can correctly calculate.

Thanks

David Stubbins
Software Engineer

E:  david.stubbins@proagrica.com<mailto:david.stubbins@proagrica.com>
W: www.proagrica.com<http://www.proagrica.com/>
T:  +44 (0)1594 545003

[cid:image001.png@01D1E1AE.32E9FE90]

From: Stubbins, David (RBI)
Sent: 07 April 2020 09:53
To: andrglo/react-window-grid <reply@reply.github.com>
Subject: RE: [andrglo/react-window-grid] ColumnHeaderRenderer (#3)

Hi there,

I have also found an issue whereby if the horizontal scrollbar is showing, then the vertical scroll is misaligned with the sticky first column when you scroll to the bottom – it looks like it isn’t taking into account the height of the bottom scrollbar to offset it.

Any ideas on how to fix this?

Thanks

David Stubbins
Software Engineer

E:  david.stubbins@proagrica.com<mailto:david.stubbins@proagrica.com>
W: www.proagrica.com<http://www.proagrica.com/>
T:  +44 (0)1594 545003

[cid:image001.png@01D1E1AE.32E9FE90]
